### ISSUE TITLE
Usability papercuts on keyboard shortcut help page

### ISSUE THREAD

**Issue by pkgw** [jupyter/notebook#603]
As a new user who was recently viewing the Notebook help page on keyboard shortcuts, I encountered two usability papercuts:

  * The shortcuts are displayed with capital letters, but they are only activated by lowercase letters. At least, that's how things are on my Linux/Firefox setup. At first I thought nothing was working because I'd by typing (say) "A" instead of plain lowercase "a".
  * The meaning of the comma syntax like "D, D" for "delete cell" is not at all clear. It took me a while to figure out that it meant I needed to press the keys twice.

**Comment by minrk** [comment]
cc @ellisonbg @captainsafia @willingc for user-facing help.

The case issue will likely change, as we are going to switch to a different system where the label will be the actual text typed, instead of the key cap ('A ≠ a', etc.).

**Comment by willingc** [comment]
@pkgw I'm sorry that you ran into these. Thank you for reporting them and your use of the word "papercuts" is brilliant.

@minrk If you can label this issue for docs, sprint friendly, I will try to have a contributor at Grace Hopper work on improving the docs. No more "papercuts". :+1:

**Comment by pkgw** [comment]
Yes, I agree that it's great terminology! I learned it [from Ubuntu](http://www.jorgecastro.org/2009/06/10/papercut-time/).

**Comment by ellisonbg** [comment]
As @minrk mentioned, we are working on a new keyboard shortcut that will
address both of these points. Will be a few months before it is live,
hopefully you can be patient with us :)

On Wed, Oct 14, 2015 at 8:37 AM, Peter Williams <notifications@github.com>
wrote:

> Yes, I agree that it's great terminology! I learned it from Ubuntu
> <http://www.jorgecastro.org/2009/06/10/papercut-time/>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jupyter/notebook/issues/603#issuecomment-148036515>.
>



-- 
Brian E. Granger
Associate Professor of Physics and Data Science
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com

**Comment by Carreau** [comment]
It's uppercase  at least on OS X because that's the convention there: 
![screen shot 2015-10-14 at 10 58 37](https://cloud.githubusercontent.com/assets/335567/10492760/9457cd1c-7262-11e5-9c3d-3d040744790e.png)

I think that being consistent with platform would be a nice thing.

**Comment by ChrisKeefe** [comment]
Just a bump for the capitalization issue. I know you all have [bigger fish to fry](https://github.com/jupyter/notebook/issues/6210), but after spending two months thinking my browser extensions were breaking the `A` and `B` shortcuts, it killed me to realize the `shift` was the problem. `A` and `a` feel fundamentally more different to programmers than they do to normal humans and this deserves attention someday.

**Comment by willingc** [comment]
Adjusted the issue tags for visibility, @ChrisKeefe

**Comment by kevin-bates** [comment]
Thanks for the label update @willingc.  I'm a little confused though regarding the combination of "enhancement" and "help wanted" on a repository in which we've been [trying to convey is in maintenance mode](https://github.com/jupyter/notebook#notice) for well over a year now.

It seems like the direction this enhancement should take is relative to Jupyter Lab since our ["bigger fish"](https://github.com/jupyter/notebook/issues/603#issuecomment-1002771781) will be based on that codebase.  Perhaps Lab already handles this condition?

If this is something trivial and zero risk (I have no idea) then perhaps it can be viewed as a usability bug rather than an enhancement.  That said, the same functionality should then be added to Lab (or RetroLab) so a regression issue is not introduced when moving to Notebook V7.

**Comment by willingc** [comment]
@kevin-bates Thanks! I removed the enhancement label. If anyone is interested in this issue, please discuss with the JupyterLab team.
